client/lib/ose-noindex/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  name: 'ose-noindex',
+
+  /**
+   * Add `<meta name="robots" content="noindex, nofollow">` to <head>.
+   * We default to enabled in all environments except production.
+   */
+  contentFor(type, appConfig) {
+    if (!appConfig.IS_PROD && type === 'head') {
```

Comment:
I don't want us to have duplicative `environment` and `IS_PROD` values. How about you copy the approach in the [source addon](https://github.com/elidupuis/ember-cli-noindex/blob/master/index.js#L15) by having an object on appConfig called `noindex` with a property called `enabled`. That way you can set it in environment.js with something like this.
```
if (environment === 'production') {
  ENV.noindex = {
    enabled: true
  }
}
```

Reply:
Ok, I have pushed changes that I believe are what you were requesting, but I am a little unclear as to how it is not just adding extra abstraction.